apps/prairielearn/src/pages/administratorInstitutions/administratorInstitutions.html.ts

```diff
@@ -78,9 +91,8 @@ export function AdministratorInstitutions({
             </div>
             <div class="card-footer">
               <small>
-                To change institutions, edit the <strong><tt>institutions</tt></strong> table. To
-                change the allowed authentication providers, edit the
-                <strong><tt>institution_authn_providers</tt></strong> table.
+                This is a list of all institutions in the database. Click on an institution name to
+                view more details or edit configuration.
```

Comment:
IMO, we can get rid of the footer entirely, I don't think this adds any information that shouldn't be clear from context.

Reply:
That makes sense. Removed in commit [ed4f9f4](https://github.com/PrairieLearn/PrairieLearn/pull/9045/commits/ed4f9f44057278fce12a5e2de1948446144ae6c6).